### DISCUSSION TITLE
turn of insert mode highlight

### DISCUSSION THREAD

**Discussion by yashamon** [nvim-lualine/lualine.nvim#897] [Configuration]
When entering insert mode the status line lights up, how to turn this off?

**Comment by yutkat** [comment]
```lua
	lualine_a = { { "mode", color = { fg = "red", bg = "grey" } } },
```

https://github.com/nvim-lualine/lualine.nvim/blob/3325d5d43a7a2bc9baeef2b7e58e1d915278beaf/doc/lualine.txt#L435-L450

**Reply by yashamon** [reply]
I don't think that answers my question, I want the same theme and colors whether the buffer is active or not, whether in insert mode or not.

**Reply by yutkat** [reply]
Unfortunately, there is no such option. I think the only solution is that you have to set the `color` option in every component.
ref: https://github.com/nvim-lualine/lualine.nvim/issues/571

**Comment by yashamon** [comment]
Thanks,  it is an acceptable work around.

**Reply by shadmansaleh** [reply]
look how https://github.com/nvim-lualine/lualine.nvim/blob/master/examples/evil_lualine.lua handles colors
